lib/web_ui/lib/src/engine/platform_views/content_manager.dart

```diff
@@ -38,6 +38,11 @@ class PlatformViewManager {
     );
   }
 
+  /// The shared instance of PlatformViewManager shared across the engine to handle
+  /// rendering of PlatformViews into the web app.
+  // TODO(dit): How to make this overridable from tests?
+  static final PlatformViewManager instance = PlatformViewManager();
```

Comment:
This is now very easy to override for tests! When we need that, this just needs to be non-final and have a setter for the testing environment. (Same as all the other singletons!)